<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CMainWindow</class>
 <widget class="QMainWindow" name="CMainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>628</width>
    <height>561</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CMainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="CMyTreeWidget" name="treeWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>591</width>
      <height>231</height>
     </rect>
    </property>
    <property name="mouseTracking">
     <bool>true</bool>
    </property>
    <property name="editTriggers">
     <set>QAbstractItemView::NoEditTriggers</set>
    </property>
    <property name="alternatingRowColors">
     <bool>true</bool>
    </property>
    <property name="columnCount">
     <number>4</number>
    </property>
    <column>
     <property name="text">
      <string notr="true">1</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string notr="true">2</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string notr="true">3</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string notr="true">4</string>
     </property>
    </column>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>20</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>PushButton</string>
    </property>
   </widget>
   <widget class="QComboBox" name="cboSortBy">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>20</y>
      <width>111</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Project</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Due date</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Priority</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>File</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="lblSortBy">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>20</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Sort by:</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>628</width>
     <height>18</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="actionOpen_file"/>
    <addaction name="actionOpen_File_Set"/>
    <addaction name="separator"/>
    <addaction name="actionSave"/>
    <addaction name="actionSave_as"/>
    <addaction name="separator"/>
    <addaction name="actionClose"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionNew">
   <property name="text">
    <string>New</string>
   </property>
  </action>
  <action name="actionOpen_file">
   <property name="text">
    <string>Open File...</string>
   </property>
  </action>
  <action name="actionOpen_File_Set">
   <property name="text">
    <string>Open File Set...</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionSave_as">
   <property name="text">
    <string>Save as...</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionClose">
   <property name="text">
    <string>Close</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>CMyTreeWidget</class>
   <extends>QTreeWidget</extends>
   <header>cmytreewidget.h</header>
   <slots>
    <slot>catchItemClicked(QTreeWidgetItem*,int)</slot>
    <slot>catchItemDoubleClicked(QTreeWidgetItem*,int)</slot>
   </slots>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>CMainWindow</receiver>
   <slot>btnTest()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>557</x>
     <y>63</y>
    </hint>
    <hint type="destinationlabel">
     <x>313</x>
     <y>280</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>treeWidget</sender>
   <signal>itemChanged(QTreeWidgetItem*,int)</signal>
   <receiver>CMainWindow</receiver>
   <slot>updateWidgets(QTreeWidgetItem*,int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>305</x>
     <y>207</y>
    </hint>
    <hint type="destinationlabel">
     <x>313</x>
     <y>280</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionNew</sender>
   <signal>triggered()</signal>
   <receiver>CMainWindow</receiver>
   <slot>actionNew()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>313</x>
     <y>280</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionClose</sender>
   <signal>triggered()</signal>
   <receiver>CMainWindow</receiver>
   <slot>actionClose()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>313</x>
     <y>280</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionExit</sender>
   <signal>triggered()</signal>
   <receiver>CMainWindow</receiver>
   <slot>actionExit()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>313</x>
     <y>280</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionOpen_file</sender>
   <signal>triggered()</signal>
   <receiver>CMainWindow</receiver>
   <slot>actionOpenFile()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>313</x>
     <y>280</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionOpen_File_Set</sender>
   <signal>triggered()</signal>
   <receiver>CMainWindow</receiver>
   <slot>actionOpenFileset()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>313</x>
     <y>280</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionSave</sender>
   <signal>triggered()</signal>
   <receiver>CMainWindow</receiver>
   <slot>actionSave()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>313</x>
     <y>280</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionSave_as</sender>
   <signal>triggered()</signal>
   <receiver>CMainWindow</receiver>
   <slot>actionSaveAs()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>313</x>
     <y>280</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>treeWidget</sender>
   <signal>itemClicked(QTreeWidgetItem*,int)</signal>
   <receiver>treeWidget</receiver>
   <slot>catchItemClicked(QTreeWidgetItem*,int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>305</x>
     <y>207</y>
    </hint>
    <hint type="destinationlabel">
     <x>305</x>
     <y>207</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>treeWidget</sender>
   <signal>itemDoubleClicked(QTreeWidgetItem*,int)</signal>
   <receiver>treeWidget</receiver>
   <slot>catchItemDoubleClicked(QTreeWidgetItem*,int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>305</x>
     <y>207</y>
    </hint>
    <hint type="destinationlabel">
     <x>305</x>
     <y>207</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>populate()</slot>
  <slot>updateWidgets(QTreeWidgetItem*,int)</slot>
  <slot>actionNew()</slot>
  <slot>actionOpenFile()</slot>
  <slot>actionOpenFileset()</slot>
  <slot>actionSave()</slot>
  <slot>actionSaveAs()</slot>
  <slot>actionClose()</slot>
  <slot>actionExit()</slot>
  <slot>btnTest()</slot>
 </slots>
</ui>
